support/azure/virtual-machines/troubleshoot-vm-boot-error.md

```diff
@@ -46,6 +46,7 @@ grub rescue>
     * [Error 15: File not found](#error15)
     * [Error: file '/boot/grub2/i386-pc/normal.mod' not found](#normal-mod-file-not-found)
     * [Error: no such partition](#no-such-partition)
+    * [Error: symbol 'grub_efi_get_secure_boot'not found](#grub_efi_get_secure_boot)
```

Comment:
Insert a space ''grub_efi_get_secure_boot' not found

Reply:
Done.